tests/runner.py

```diff
@@ -81,6 +80,7 @@
   'posixtest_browser',
   'minimal0',
   'wasmfs',
+  'wasm64',
```

Comment:
I was not aware of the random100 thing..  out of interest why make it random?  Why not have a fixed "top 100 smoke test"?

Also, why not include "other" in "core_test_modes" ?   Should we rename "core_test_modes" to "expected_pass_modes"?

Reply:
Should asan and lsan now be moved up since they are now expected to always pass?  (We run them on the linux-testsuite builder continuously).

Reply:
Randomness was just a simple thing to get running. Picking a fixed smoke test suite takes work :smile: and it gets out of date.

Good points. I'll rename to `passing_core_test_modes` and `misc_test_modes` and add some comments. Things like `other`, `browser`, etc. should not be here, I think as they are not variations on the core tests - the random test picker selects from `test_core.py` by default. I'll clarify in a comment.

I see we also have some existing docs here: https://emscripten.org/docs/getting_started/test-suite.html#running-a-bunch-of-random-tests